### DISCUSSION TITLE
【リリースノート】過去のPRに付けたラベルは適切でしょうか？？

### DISCUSSION THREAD

**Discussion by kazzyfrog** [first-contributions-ja/first-contributions-ja.github.io#169] [10: 技術についての質問] [answered]
生成したリリースノート：https://github.com/first-contributions-ja/first-contributions-ja.github.io/releases

こちらを生成するにあたって、過去のPRにラベルをつけました！

以下など？を確認・検討していければと思います！

1. styleやrefactorは必要か
2. fixとbugFixの方が良いか
3. コード関連と、その他でまとめた方がいいか
4. 過去のPRのラベル付けが適切か

問題なければ、
「バージョンタグとリリースノート」に関するイシューは、完了します！！

**Comment by kagomen** [comment, marked as answer]
リリースノート作成とラベル分け、ありがとうございます！🙏

1. styleやrefactorは必要か
  -> style も refactor も現時点では不要かもしれませんが、v2の実装で必要になってきそうです
1. fixとbugFixの方が良いか
 -> fix 以外に、純粋な変更につけられる prefix があったら便利だなと思いました
1.  コード関連と、その他でまとめた方がいいか
 -> こちらは今のままで良いなと思いました！
1.  過去のPRのラベル付けが適切か
 -> 以下に気になった振り分けを抜き出しました

🐛 Bug Fix
- トップページとREADMEのサイト名をFirst Contributions JAに変更 -> ???
- Contributors.jsonの、データの内容を更新 -> ???

📚 Documentation
- イシューテンプレートの追加 -> chore ? feat ?
- pull requestテンプレートの追加 -> chore ? feat ?
- イシューテンプレートの追加・削除  -> chore ? 

🎨 Style
- レスポンシブデザインの実装 -> feat ?
- fix/font -> chore ?
- NotoSansJPのフォント設定の修正 -> fix ? chore ?

***

特に以下の振り分けが難しいなと思いました。

> - トップページとREADMEのサイト名をFirst Contributions JAに変更 -> ???
> - Contributors.jsonの、データの内容を更新 -> ???

2で、

> fix 以外に純粋な変更につけられる prefix があったら便利

と述べた理由でもあります。
これって @kazzyfrog さんだったら何に振り分けしますか？👀

**Reply by kazzyfrog** [reply]
ありがとうございます！
そしたら、下記については、一旦現状のまま行こうと思います！
> 1.styleやrefactorは必要か
3.コード関連と、その他でまとめた方がいいか

---

> 何に振り分けしますか？👀

自分は、`fix`かなと思いました！
バグ修正というよりは、仕様変更に伴う修正というニュアンスが近いかもです！
（結局これも、放置したら不具合になると思うので、）それも`fix`でいいかもと思った次第です！

**Reply by kazzyfrog** [reply]
イシューと、プルリクテンプレに関して、
これは、Next.jsでのアプリ（サイト）に関する機能や設定ではなく、リポジトリというかGitHubの設定なので、
READMEやCONTRIBUTING.mdと同じような扱い（必要だけど、アプリのchoreとも違う）でいいのかなと思い、docsにしました！

Styleは、「バグでも機能追加でもない、見た目のみの変更」という感じなので、
「レスポンシブデザインの実装 」は、featかstyleのどっちかですね！featの方がいいかも！

フォントは、styleかなと思いました！
Next.jsのフォント最適化の機能を使っているのがややこしいですが、
設定しやすくしているだけで、結局あれもCSSに適用されると思うので！👍

**Reply by kazzyfrog** [reply]
「レスポンシブデザインの実装 」は、featに変更しました！！
リリースノートも、後ほど生成し直します！

**Reply by kagomen** [reply]
- fix
> 放置したら不具合になると思う
- docs
> アプリのchoreとも違う
- style
> Next.jsのフォント最適化の機能を使っているのがややこしいですが、
設定しやすくしているだけで、結局あれもCSSに適用される

こちら、理解しました！
明快にご説明していただき、ありがとうございます…！🙏
